RELEASENOTES.md

```diff
@@ -20,6 +20,8 @@
 -->
 # Cordova-node-xcode Release Notes
 
+* Updated `objects = {` to be followed by a blank line, matching current Xcode layout ([#46](https://github.com/apache/cordova-node-xcode/pull/46))
```

Comment:
```suggestion
```

Reply:
@fbartho can you remove this line?

I think it would be better if we can do the release notes for all changes at once, when we make the release.

I think the other changes should be good to go. I did see the extra blank line that is introduced by Xcode 10.1 if I would rename a file. Thanks again.